### DISCUSSION TITLE
[resolu] Readme du Template site-static-v2

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#3] [General] [answered]
hello @hrenaud,
Je crois qu'il manque un mot ici : 
https://github.com/cnumr/fiches-wordpress-greenit/blob/f515bc50f5a306489ac8464c570c4cc6843ed980/best-practices-WordPress/template/README.md?plain=1#L15

**Comment by hrenaud** [comment, marked as answer]
fix dans le comit https://github.com/cnumr/fiches-wordpress-greenit/commit/85a7e9faa280a0fa56a5755951a3edb014b13a21
